### ISSUE TITLE
Tech’s Next Big Task: Taking the Office Water Cooler Virtual

### ISSUE THREAD

**Issue by maherou** [Collegeville/VirtualTeams#44]
Link: https://www.wsj.com/articles/techs-next-big-task-taking-the-office-water-cooler-virtual-11599397201

Summary: Innovators in the remote-work space are rushing to build and modify digital services that help people feel like they are working together when they are miles apart.

Discussion:  This article describes experiences and struggles that teams have been having in the six months since the pandemic started.  In particular it focuses on how teams are trying to enable the incidental interactions that are hard to replace in a virtual world.  Some techniques discussed are the virtual hallway, always-on virtual presence, random pairing of members of an organization.  

The overall tone of the article, in my estimation, is that we have not adequately figured out a virtual water cooler kind of serendipitous interaction so far.

**Comment by amoralesg001** [comment]
I agree with you on the overall tone of the article. It seems as if these technologies are not the full solution for remote teams, but they definitely can help. I think this quote reflects the message well: "they're (these technologies like asprin for the headache". 

I have heard of Donut, Miro, and Microsoft's software that they use in previous articles. This issue #35 speaks about new forms of technologies Microsoft is designing for Virtual environments (one of which the article speaks about). I think these technologies are the step to the right direction. I also find this area extremely interesting since new research is coming out everyday about collaboration while working emotely. 

I also thought that this article emphasized the importance of reestablishing "togetherness" while working remotely. Since working remotely can be lonely (especially because of Covid), it seems that many companies are finding new ways to decrease loneliness in their company.

**Comment by elaineraybourn** [comment]
I was unable to read the WSJ link so I found this version that does not require a subscription:
https://www.techmaq.com/techs-next-big-task-taking-the-office-water-cooler-virtual/.

This new emphasis on the topic of "presence" in CS literature is cool to see. There is a lot of literature on this topic, and some of the "new" approaches to avatars and virtual meetings have been investigated by researchers for decades. I'd be curious to know what users think about Donut. My own research conducted with British Telecom (BT) in early 2000 suggests that these pop up conversations don't work unless the interlocutors already know each other or have a conversation icebreaker: 

[Adding cultural signposts in adaptive community-based virtual environments](https://ieeexplore.ieee.org/abstract/document/8155973)
E. M. Raybourn, N. Kings, J. Davies
